.github/workflows/ci.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
       - name: Checkov scan (with JUnit output)
         run: |
           # Checkov with JUnit XML output
-          npx nx affected -t checkov --skip-check CKV_AWS_73,CKV_AWS_115,CKV_AWS_116,CKV_AWS_117,CKV_AWS_120 --external-checks-dir ./.checkov --output junitxml --output-file-path reports/infrastructure/checkov-results.xml
+          npx nx affected -t checkov --skip-check CKV_AWS_73,CKV_AWS_115,CKV_AWS_116,CKV_AWS_117,CKV_AWS_120 --external-checks-dir ./.checkov --output junitxml --output-file-path console,reports/infrastructure
```

Comment:
Added console so output also included in stdout ([see](https://www.checkov.io/2.Basics/CLI%20Command%20Reference.html#:~:text=the%20CLI%20output%20will%20be%20printed%20to%20the%20console))